web/src/components/table/use-cases/sessions.tsx

```diff
@@ -189,6 +201,34 @@ export default function SessionsTable({
       enableHiding: true,
       enableSorting: true,
     },
+    {
+      accessorKey: "inputCost",
+      id: "inputCost",
+      header: "Input Cost",
+      enableHiding: true,
+      defaultHidden: true,
+      cell: ({ row }) => {
+        const value: number | undefined = row.getValue("inputCost");
+
+        return value !== undefined ? (
+          <span>{usdFormatter(value, 2, 2)}</span>
+        ) : undefined;
+      },
+    },
+    {
+      accessorKey: "outputCost",
+      id: "outputCost",
+      header: "Output Cost",
+      enableHiding: true,
+      defaultHidden: true,
+      cell: ({ row }) => {
+        const value: number | undefined = row.getValue("outputCost");
+
+        return value !== undefined ? (
+          <span>{usdFormatter(value, 2, 2)}</span>
```

Comment:
lets round here the same way we do on the generations table: https://github.com/langfuse/langfuse/blob/richard/lfe-461/web/src/components/table/use-cases/generations.tsx#L309